Readme.md

```diff
@@ -1,22 +1,30 @@
 <!-- default badges list -->
-![](https://img.shields.io/endpoint?url=https://codecentral.devexpress.com/api/v1/VersionRange/128540369/13.1.8%2B)
 [![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/E4970)
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
-<!-- default file list -->
-*Files to look at*:
-
-* [Default.aspx](./CS/WebSite/Default.aspx) (VB: [Default.aspx](./VB/WebSite/Default.aspx))
-* [Default.aspx.cs](./CS/WebSite/Default.aspx.cs) (VB: [Default.aspx.vb](./VB/WebSite/Default.aspx.vb))
-<!-- default file list end -->
-# How to get a value of a ASPxComboBox hidden column on the client side
+# Combo Box for ASP.NET Web Forms - How to get hidden column values on the client
 <!-- run online -->
 **[[Run Online]](https://codecentral.devexpress.com/e4970/)**
 <!-- run online end -->
 
+This example demonstrates how to get values stored in the [Combo Box](http://docs.devexpress.devx/AspNet/11653/components/data-editors/combobox) control's hidden column on the client.
```

Comment:
Just to satisfy my curiosity -- how many hidden columns the Combo Box may have? I suspect that any number of columns -- more than one, at the very least. And the described technique allows one to get values stored in any of them. I'm only suggesting that another wording probably would have been more efficient in conveying this idea:

```suggestion
This example demonstrates how to get values stored in a hidden column of the [Combo Box](http://docs.devexpress.devx/AspNet/11653/components/data-editors/combobox) control on the client.
```

**This is an optional suggestion**. If you believe that the original version is better, feel free to disagree.